pkg/scheduler/framework/plugins/interpodaffinity/plugin.go

```diff
@@ -132,3 +134,88 @@ func GetNamespaceLabelsSnapshot(logger klog.Logger, ns string, nsLister listersv
 	logger.V(3).Info("getting namespace, assuming empty set of namespace labels", "namespace", ns, "err", err)
 	return
 }
+
+func (pl *InterPodAffinity) isSchedulableAfterPodChange(logger klog.Logger, pod *v1.Pod, oldObj, newObj interface{}) (framework.QueueingHint, error) {
```

Comment:
Can we take `Namespace` selector into consideration?
https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#namespace-selector

Reply:
I dont think we should do this because `podMatchesAllAffinityTerms` has implemented it.
https://github.com/kubernetes/kubernetes/blob/b529e6ff1c7edb3bb1eaca185b9662d8e42ac2d0/pkg/scheduler/framework/plugins/interpodaffinity/filtering.go#L145